code/drasil-example/README.md

```diff
@@ -7,33 +7,51 @@ This folder contains our working examples (implementations in Drasil of the prev
 
 **dblpend**
   - Contains the current state of the Double Pendulum case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [dblpend](https://github.com/Zhang-Zhi-ZZ/CAS741Project) Double Pendulum example.
 
 **gamephysics**
   - Contains the current state of the Game Physics case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [gamephysics](https://github.com/smiths/caseStudies/tree/master/CaseStudies/gamephys) 2D Rigid Body Physics Library.
 
 **glassbr**
   - Contains the current state of the GlassBR case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [Glass-BR](https://github.com/smiths/caseStudies/tree/master/CaseStudies/glass) example.
   
 **hghc**
   - Contains the current state of the HGHC (fuel pin) example implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [hghc](https://github.com/smiths/caseStudies/tree/master/CaseStudies/hghc) toy example.
   
 **swhsnopcm**
   - Contains the current state of the solar water heating system case study implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [Nopcm](https://github.com/smiths/caseStudies/tree/master/CaseStudies/noPCM) SWHS without PCM example.
 
 **pdcontroller**
   - Contains the current state of the PD Controller example implemented in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [Proportional Derivative Controller](https://github.com/muralidn/CAS741-Fall20) example.
 
 **projectile**
   - Contains the current work on implementing the projectile example in Drasil
+  - This is a recreation of the original, manually-created version 
+    in the [Projectile](https://github.com/smiths/caseStudies/tree/master/CaseStudies/projectile) motion analysis example.  
```

Comment:
There are multiple versions of projectile.  You should point to the following version:

https://github.com/smiths/caseStudies/tree/master/CaseStudies/projectile/projectileSRS